.github/workflows/triage.yml

```diff
@@ -135,7 +135,7 @@ jobs:
               content: system "swift", "build"
 
             - label: long build
-              path: Formula/(agda|arangodb|boost|deno|gcc|ghc|llvm|node|ponyc|rust|swift|texlive|qt)(@[0-9]+)?.rb
+              path: Formula/(agda|arangodb|boost|deno|dotnet|emscripten|gcc|ghc|llvm|node|ponyc|rust|swift|texlive|qt)(@[0-9]+)?.rb
```

Comment:
Feels like this might be nice to end up as a list/array?

Reply:
That would be much nicer, yes. I think we need to extend our PR label action to support that, though. (And my JavaScript is very bad...)

Maybe @dawidd6 or @FnControlOption wouldn't mind taking a look?